src/modules/ecdh/tests_impl.h

```diff
@@ -123,10 +123,40 @@ void test_bad_scalar(void) {
     CHECK(secp256k1_ecdh(ctx, output, &point, s_overflow, ecdh_hash_function_test_fail, NULL) == 0);
 }
 
+void test_result_basepoint(void) {
+    secp256k1_pubkey point;
+    secp256k1_scalar rand;
+    unsigned char s[32];
+    unsigned char s_inv[32];
+    unsigned char out[32];
+    unsigned char out_eq[32];
+    int i;
+
+    for (i = 0; i < 64; i++) {
```

Comment:
This should depend on `count`

Reply:
`test_ecdh_generator_basepoint` above also has a similar issue, right?

Reply:
No, it turns out that's a known-output test with a fixed output.

Reply:
> No, it turns out that's a known-output test with a fixed output.

Nevermind, I misremembered this. I saw a test where I assumed it should depend on `count` but it turned out to be a known-output test. But it wasn't this one.  

Fixed.